scripts/executors/src/start.ts

```diff
@@ -210,8 +210,19 @@ async function getSelectedTarget(
     .filter(targetName => {
       return omitTargets.includes(targetName) ? false : true;
     })
+    .map(targetName => {
+      const targetConfigurations = Object.keys(projectTargets[targetName].configurations || {});
```

Comment:
unwrapping configurations into separate targets within the CLI

<img width="214" alt="image" src="https://github.com/user-attachments/assets/96dc6cdb-5e70-46d4-b048-a48de24ad763">